utils/theme.js

```diff
@@ -0,0 +1,32 @@
+const chalk = require("chalk");
+
+const { loadConfig } = require('./common');
+
+const message = (color, output) => console.log(chalk.keyword(color)(...output));
+
+const defaultTheme = {
+  success: (...output) => message('green', output),
+  warning: (...output) => message('orange', output),
+  error: (...output) => message('red', output),
```

Comment:
It would be great if `defaultTheme` supports `hint` which can be utilized [here](https://github.com/theenadayalank/lint-prepush/blob/master/utils/execTask.js#L19)